core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala

```diff
@@ -70,29 +70,60 @@ object Runtime {
 }
 
 /** Runtime status enum */
-sealed trait RuntimeStatus extends EnumEntry with Product with Serializable
+sealed trait RuntimeStatus extends EnumEntry with Product with Serializable {
+  // end status if it is a transition status. For instance, terminalStatus for `Starting` is `Running`
+  def terminalStatus: Option[RuntimeStatus]
```

Comment:
right now, `terminalStatus` is only used to make log message a bit more informative